aptos-move/framework/aptos-framework/sources/object.move

```diff
@@ -210,23 +209,24 @@ module aptos_framework::object {
         create_object_internal(creator_address, obj_addr, false)
     }
 
-    /// Create a new object from a GUID generated by an account.
-    public fun create_object_from_account(creator: &signer): ConstructorRef {
-        let guid = account::create_guid(creator);
-        create_object_from_guid(signer::address_of(creator), guid)
+    /// Create a new object by generating a random UUID based on transaction hash.
+    /// The created object is deletable as we can guarantee the same UUID can 
+    /// never be regenerated with future txs.
+    public fun create_object(creator: &signer): ConstructorRef {
+        let uuid = aptos_framework::transaction_context::create_uuid();
+        create_object_internal(signer::address_of(creator), uuid, true)
     }
 
-    /// Create a new object from a GUID generated by an object.
-    public fun create_object_from_object(creator: &signer): ConstructorRef acquires ObjectCore {
-        let guid = create_guid(creator);
-        create_object_from_guid(signer::address_of(creator), guid)
+    /// Deprectated. 
+    /// Create a new object by generating a random UUID based on transaction hash.
+    public fun create_object_from_account(creator: &signer): ConstructorRef {
+        create_object(creator)
     }
 
-    fun create_object_from_guid(creator_address: address, guid: guid::GUID): ConstructorRef {
-        let bytes = bcs::to_bytes(&guid);
-        vector::push_back(&mut bytes, OBJECT_FROM_GUID_ADDRESS_SCHEME);
-        let obj_addr = from_bcs::to_address(hash::sha3_256(bytes));
-        create_object_internal(creator_address, obj_addr, true)
+    /// Deprectated. 
```

Comment:
Typo: Should be "Deprecated"

Reply:
Also, how come **both** `create_object_from_account` and `create_object_from_object` are marked as "deprecated"?

Did you mean to only mark one of them?

Reply:
Or have they both been deprecated by `create_object`.

If so, please update comments to indicate which function they are deprecated by (i.e., which function should be called instead)

Reply:
Both the functions are deprecated by `create_object`. I will remove "deprecated" comment for now.

Reply:
They're not really deprecated as they offer alternatives for creating objects. We need to more deeply evaluate if they no longer have any useful purposes